src/svg-to-vectordrawable.js

```diff
@@ -407,6 +407,12 @@ JS2XML.prototype.refactorData = function(data, floatPrecision, fillBlack, tint)
                 }
             }
             elem.addAttr({ name: 'd', value: pathData, prefix: '', local: 'd' });
+            elem.removeAttr('x')
```

Comment:
Here we need to remove the original attributes after converting them to the `d` attribute.